src/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -761,14 +762,8 @@ private void notifyCasualties(final GUID battleID, final IDelegateBridge bridge,
       public void run() {
         try {
           getRemote(firingPlayer, bridge).confirmEnemyCasualties(battleID, "Press space to continue", hitPlayer);
-        } catch (final ConnectionLostException cle) {
-          // somone else will deal with this
-          // System.out.println(cle.getMessage());
-          // cle.printStackTrace(System.out);
-        } catch (final GameOverException e) {
-          // ignore
         } catch (final Exception e) {
-          // ignore
+          ClientLogger.logQuietly(e);
```

Comment:
Any reason to add a logger here but not some of the other exceptions you merged that are empty? Also if you are going to add a logger here then probably should address @RoiEXLab points about replacing printStackTrace(). Prefer if we either don't change functionality at all or do it consistently across the PR.